### ISSUE TITLE
Minihtml "Definition & Reference" hover for sublime-syntax

### ISSUE THREAD

**Issue by michaelblyons** [SublimeText/PackageDev#279]
Is there a way to get this thing for named contexts in `.sublime-syntax`? Ideally, I'd like the hover to be available (and to include) both definitions and uses... maybe even across files? (e.g. Git Formats/Git Common)

![Screen Shot 2020-02-03 at 5 14 54 PM](https://user-images.githubusercontent.com/2701605/73695938-0f7b3d00-46d3-11ea-99e8-ccceb8f811c9.png)

**Comment by FichteFoll** [comment]
Yes and no. Yes, it is possible, but I would need to remove the string scopes because ST explicitly excludes them currently.

Upstream issue: https://github.com/sublimehq/sublime_text/issues/2546

**Comment by michaelblyons** [comment]
Thanks. That's exactly what I was looking for.

**Comment by michaelblyons** [comment]
Lurkers, you can hack in the "uses" hover on a context definition by dropping the string scope in `context_definition`'s `set`:

https://github.com/SublimeText/PackageDev/blob/ec462749684d8b4a5cd4e362639007d339e72bb7/Package/Sublime%20Text%20Syntax%20Definition/Sublime%20Text%20Syntax%20Definition.sublime-syntax#L219

You are also able to F12 to definitions from `include`s, but I haven't seen the hover on them.
